clickhouse-admin/types/src/lib.rs

```diff
@@ -501,6 +502,404 @@ impl RaftConfig {
     }
 }
 
+// While we generally use "Config", in this case we use "Conf"
+// as it is the four letter word command we are invoking:
+// `clickhouse keeper-client --q conf`
+/// Keeper configuration information
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct KeeperConf {
+    /// Unique server id, each participant of the ClickHouse Keeper cluster must
+    /// have a unique number (1, 2, 3, and so on).
+    pub server_id: KeeperId,
+    /// Whether Ipv6 is enabled.
+    pub enable_ipv6: bool,
+    /// Port for a client to connect.
+    pub tcp_port: u16,
+    /// Allolw list of 4lw commands.
```

Comment:
```suggestion
    /// Allow list of 4lw commands.
```

Reply:
arrgghhhhhh I merged without fixing this typo, I'll open up a little PR with the typo fix